src/Umbraco.Core/Logging/ILogger.cs

```diff
@@ -25,158 +27,67 @@ public interface ILogger
-        /// <param name="message">A message.</param>
-        void Error(Type reporting, Exception exception, string message);
-
-        /// <summary>
-        /// Logs an error exception.
-        /// </summary>
-        /// <param name="reporting">The reporting type.</param>
-        /// <param name="exception">An exception.</param>
-        /// <remarks>The message string is unspecified and is implementation-specific.</remarks>
-        void Error(Type reporting, Exception exception);
-
-        /// <summary>
-        /// Logs an error message.
-        /// </summary>
-        /// <param name="reporting">The reporting type.</param>
-        /// <param name="message">A message.</param>
-        void Error(Type reporting, string message);
+        void LogCritical(string messageTemplate, params object[] propertyValues);
 
         /// <summary>
         /// Logs an error message with an exception.
         /// </summary>
-        /// <param name="reporting">The reporting type.</param>
         /// <param name="exception">An exception.</param>
         /// <param name="messageTemplate">A message template.</param>
         /// <param name="propertyValues">Property values.</param>
-        void Error(Type reporting, Exception exception, string messageTemplate, params object[] propertyValues);
+        void LogError(Exception exception, string messageTemplate, params object[] propertyValues);
 
         /// <summary>
         /// Logs an error message.
         /// </summary>
-        /// <param name="reporting">The reporting type.</param>
         /// <param name="messageTemplate">A message template.</param>
         /// <param name="propertyValues">Property values.</param>
-        void Error(Type reporting, string messageTemplate, params object[] propertyValues);
-
-        /// <summary>
-        /// Logs a warning message.
-        /// </summary>
-        /// <param name="reporting">The reporting type.</param>
-        /// <param name="message">A message.</param>
-        void Warn(Type reporting, string message);
+        void LogError(string messageTemplate, params object[] propertyValues);
 
         /// <summary>
         /// Logs a warning message.
         /// </summary>
-        /// <param name="reporting">The reporting type.</param>
         /// <param name="messageTemplate">A message template.</param>
         /// <param name="propertyValues">Property values.</param>
-        void Warn(Type reporting, string messageTemplate, params object[] propertyValues);
-
-        /// <summary>
-        /// Logs a warning message with an exception.
-        /// </summary>
-        /// <param name="reporting">The reporting type.</param>
-        /// <param name="exception">An exception.</param>
-        /// <param name="message">A message.</param>
-        void Warn(Type reporting, Exception exception, string message);
+        void LogWarning(string messageTemplate, params object[] propertyValues);
```

Comment:
The code should avoid the use of params keyword as it causes an array allocation. It would be better to also include overloads for 1, 2,3 and 4 parameters without the use of the params key.